### PR TITLE
Allow Bard starting items in `--diablo`

### LINKED ISSUE #1324: [diablo] Bard does not receive a weapon in the new game
Bard has no weapon if you start a game with a new character in Diablo.
bard2

### PR DESCRIPTION
**Pull request by glebm**
Fixes #1324

### CODE REVIEW

File: SourceX/DiabloUI/selhero.cpp [hunk 1]
```diff
@@ -358,7 +358,7 @@ static bool shouldPrefillHeroName()
 void selhero_ClassSelector_Select(int value)
 {
 	int hClass = vecSelHeroDlgItems[value]->m_value;
-	if (gbSpawned && (hClass == PC_ROGUE || hClass == PC_SORCERER || hClass == PC_BARD)) {
+	if (gbSpawned && hClass != PC_WARRIOR) {
```
Comment (glebm) [inline SourceX/DiabloUI/selhero.cpp:361]: This is just an inconsistency I noticed

Comment (AJenbo) [inline SourceX/DiabloUI/selhero.cpp:361]:
```suggestion
	if (gbSpawned && hClass != PC_WARRIOR && hClass != PC_MONK) {

```

Comment (AJenbo) [reply SourceX/DiabloUI/selhero.cpp]: What's the inconsistency?

Comment (glebm) [reply SourceX/DiabloUI/selhero.cpp]: `PC_BARBARIAN` was allowed

Comment (AJenbo) [reply SourceX/DiabloUI/selhero.cpp]: Oh, the barbarian, good catch :)

Comment (AJenbo) [reply SourceX/DiabloUI/selhero.cpp]: Ha ha, just realized this wasn't an oversight :D

**Pull request merged**